### ISSUE TITLE
Custom variants does not work anymore

### ISSUE THREAD

**Issue by kikoanis** [iamhosseindhv/notistack#251]
It seems that something has changed in the last release that prevent custom variants (custom classes) from implementing. I looked at the changes and I could not put my finger on what has changed. This is related to release 0.9.10

implementing something like this, does not work any more
```
<SnackbarProvider
    classes={{
        variantSuccess: classes.success,
        variantError: classes.error,
        variantWarning: classes.warning,
        variantInfo: classes.info,
    }}
>
 ```

**Comment by iamhosseindhv** [comment]
Could you provide a reproduction?
Here's a working example with `v0.9.10`.
https://codesandbox.io/s/friendly-wilson-dwniy?file=/App.js

**Comment by kikoanis** [comment]
oh does it? It used to. The example you provided does not work as expected anymore. Clicked on success (the one that is supposed to be overridden) but still shows the original background color. 

What am I missing?

**Comment by picosam** [comment]
I confirm. It seems to be broken in `v0.9.10`.

**Comment by iamhosseindhv** [comment]
You're right. Bug is fixed and published in `v0.9.11`. Appreciate if you could confirm the fix.

**Comment by kikoanis** [comment]
I can confirm the fix is working fine. Thanks for the quick response and the fix.
Great library BTW.
